hadoop-cloud-storage-project/hadoop-cos/src/main/java/org/apache/hadoop/fs/cosn/BufferPool.java

```diff
@@ -86,10 +85,6 @@ private File createDir(String dirPath) throws IOException {
       } else {
         LOG.debug("buffer dir: {} already exists.", dirPath);
       }
-    } else {
-      throw new IOException("creating buffer dir: " + dir.getAbsolutePath()
-          + "unsuccessfully.");
-    }
```

Comment:
Would you fix indent?

Reply:
Thanks @aajisaka for the review.
Yes I would fix it, and ran the test on this issue.